apps/example/src/index.ts

```diff
@@ -73,7 +118,12 @@ new Yab()
 	)
 	.use(statics("/public", { assetsDir: "./public" }))
 	// .use(rateLimit("redis", {}))
-	.use(router("/api", [UserController]))
+	.use(
+		router("/api", [UserController], {
+			customValidation: async (schema: any, payload: any) => {},
```

Comment:
if user add custom validation that doing nothing. why the validation still working? it should be pass everything